easybuild/easyconfigs/q/qmflows/qmflows-0.12.1-foss-2021a.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'PythonBundle'
+
+name = 'qmflows'
+version = '0.12.1'
+
+homepage = 'https://github.com/SCM-NV/qmflows'
+description = """This library tackles the construction and efficient execution 
+of computational chemistry workflows. This allows computational chemists to use 
+the emerging massively parallel compute environments in an easy manner and focus 
+on interpretation of scientific data rather than on tedious job submission 
+procedures and manual data processing.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = {
+    ('Python', '3.9.5'),
+    ('RDKit', '2021.03.4'),
+    ('HDF5', '1.10.7'),
+    ('SciPy-bundle', '2021.05'),
+    ('Rust', '1.52.1')
```

Comment:
This dependency on Rust will not be needed if you drop the extension of `cryptography` as suggested in a previous comment